lib/ui/profile/interactor/profile_bloc.freezed.dart

```diff
@@ -1386,6 +1386,7 @@ mixin _$ProfileState {
   PageState get pageState => throw _privateConstructorUsedError;
   PageCommand? get command => throw _privateConstructorUsedError;
   ProfileData? get profileData => throw _privateConstructorUsedError;
+  bool get doesNotHaveProfile => throw _privateConstructorUsedError;
```

Comment:
doesNotHaveProfile => hasProfile i think its a better name for this , not a blocker , just a nit

Reply:
I tried changing it and realized why it should be as it is, if not we use the negative everywhere, like !hasProfile, which is even more confusing in the code... leaving this.